src/gleam/erlang/file.gleam

```diff
@@ -109,8 +109,97 @@ pub type Reason {
   Exdev
   /// File was requested to be read as UTF-8, but is not UTF-8 encoded.
   NotUTF8
+  /// Path is not a directory
+  Badarg
 }
 
+/// Returns true if path refers to a directory, otherwise false.
+///
+/// ## Examples
+///
+///    > is_dir("/tmp")
+///    True
+///
+///    > is_dir("resume.pdf")
+///    False
+pub external fn is_dir(path: String) -> Bool =
```

Comment:
Could we use `directory` rather than `dir` in all these functions? In the stdlib we've avoided abbreviations so far.

Reply:
Done